### PR TITLE
move invokers into docs

### PR DESCRIPTION
**Pull request by jldec**
- copied invoker content from GitHub readme's (still need examples)
- added to docs navigation sidebar
- tweaked links in header

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -18,3 +18,18 @@ docs:
         url: /docs/getting-started-on-gke/
       - title: "with RBAC"
         url: /docs/running-on-gke-with-rbac/
+  - title: Invokers
+    url: /invokers/
+    children:
+      - title: "Command"
```
Comment (scothis) [inline _data/navigation.yml:24]: Can we derive this list from the collection? It's annoying to need to update the nav every time we add a new invoker.

File: _invokers/command.md [hunk 2]
```diff
@@ -12,4 +12,121 @@ releases: [
 ]
 ---
 
-<!-- TODO write something meaningful -->
+The command function invoker provides a Docker base layer for a function consisting of a single command.
+It accepts gRPC requests, invokes the command for each request in the input stream,
+and sends the command's output to the stream of gRPC responses.
+
+## Install as a riff invoker
```
Comment (scothis) [inline _invokers/command.md:19]: Installation should be handled by the invoker layout. Including it again will lead to confusion.

File: _invokers/command.md [hunk 3]
```diff
@@ -12,4 +12,121 @@ releases: [
...
+
+```bash
+riff invokers apply -f https://github.com/projectriff/command-function-invoker/raw/master/command-invoker.yaml
+```
+
+## riff Commands
```
Comment (scothis) [inline _invokers/command.md:25]: I'm hesitant to include the riff-cli docs unless we also have a mechanism to keep them up to date.

File: _config.yml [hunk 4]
```diff
@@ -45,7 +44,13 @@ defaults:
       path: ""
       type: invokers
     values:
-      layout: invoker
+      layout: single
```
Comment (scothis) [inline _config.yml:47]: It's important for invokers to use the invokers layout. We can change the invokers layout to embed within the docs single layout.

**Review by scothis — CHANGES_REQUESTED** [review]

File: _data/navigation.yml (hunk 1 above)
Comment (jldec) [reply _data/navigation.yml:24]: yeah, agree, but autocomputing the sidebar nav from multiple collections requires liquid tag gymnastics that i'm a bit reluctant to spend time on - especially considering that this jekyll docs theme is temporary/throwaway - open to suggestions tho if you know how.

File: _invokers/command.md (hunk 2 above)
Comment (jldec) [reply _invokers/command.md:19]: Actually i think limiting each page to a single markdown content blob is simpler and allows us to put more important things at the top (just like the README on github). I'd like to to work on the content first, with examples, and then decide if we want to formalize the installation bits in the template.

File: _invokers/command.md (hunk 3 above)
Comment (jldec) [reply _invokers/command.md:25]: ok - agree - will remove.

File: _config.yml (hunk 4 above)
Comment (jldec) [reply _config.yml:47]: will reconsider after working on the content first.

**Pull request merged**